modules/apps/portal/portal-inactive-request-handler/src/main/java/com/liferay/portal/inactive/request/handler/internal/InactiveRequestHandlerImpl.java

```diff
@@ -67,15 +65,7 @@ public void processInactiveRequest(
 		PrintWriter printWriter = response.getWriter();
 
 		if (!_showInactiveRequestMessage) {
-			try {
-				_portal.sendError(
-					HttpServletResponse.SC_NOT_FOUND,
-					new NoSuchLayoutException(),
-					_portal.getOriginalServletRequest(request), response);
-			}
-			catch (ServletException se) {
-				throw new IOException(se);
-			}
+			printWriter.print(StringPool.BLANK);
```

Comment:
Why a blank String, this is essentially not writing anything. If this is what we want, why not just not to do anything?

Reply:
If it's the same result then I have no problem just removing this.